nixos/modules/installer/sd-card/sd-image.nix

```diff
@@ -33,6 +32,15 @@ in
   ];
 
   options.sdImage = {
+    makeFsExtraArgs = mkOption {
```

Comment:
Just call this `makeFsArgs` I think?

Reply:
This exposes an internal API as-is externally. This shouldn't be done IMO.

Reply:
Ok, I removed it.